packages/protocol/script/test_deploy_on_l1.sh

```diff
@@ -6,7 +6,7 @@ set -e
 PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 \
 SHARED_SIGNAL_SERVICE=0x0000000000000000000000000000000000000000 \
 PROPOSER=0x0000000000000000000000000000000000000000 \
-GUARDIAN_PROVER=0xa0Ee7A142d267C1f36714E4a8F75612F20a79720 \
+GUARDIAN_PROVERS="0x1000777700000000000000000000000000000001,0x1000777700000000000000000000000000000002,0x1000777700000000000000000000000000000003,0x1000777700000000000000000000000000000004,0x1000777700000000000000000000000000000005" \
```

Comment:
Aren't they contract addresses ? 🤔

Reply:
Its ok since those addresses are only for testing the deployment